Src/GoogleApis.Auth.DotNet4/Properties/AssemblyInfo.cs

```diff
@@ -28,8 +28,14 @@
 [assembly: AssemblyCopyright("Copyright © Google Inc 2013")]
 [assembly: AssemblyTrademark("")]
 [assembly: AssemblyCulture("")]
-// Google.Apis.Auth.DotNet4.Tests won't compile in "ReleaseSigned" configuration.
-#if DEBUG
+
+#if SIGNED
+[assembly: InternalsVisibleTo("Google.Apis.Auth.DotNet4.Tests,PublicKey=" +

```

Comment:
Hi, it turned out there were still some pieces missing and that's why visual studio didn't complain about using the wrong public key here.
Fixed the public key.